src/api/app/models/review.rb

```diff
@@ -204,6 +215,24 @@ def create_notification(params = {})
     # send email later
     Event::ReviewWanted.create params
   end
+
+  private
+
+  # Note that this is not an polymorphic association at the moment as it is possible
```

Comment:
English correction: should be `not a polymorphic`

Also I dont think its really correct to say `this is not an polymorphic association`, because `self.entity` is still a polymorphic association its just not authoritative.